RELEASE_SCHEDULE.md

```diff
@@ -9,7 +9,7 @@
 
 ## Release Schedule
 
-We aim to push a new release of Prebid.js every week on Tuesday.
+We aim to push a new release of Prebid.js every week by Thursday.
```

Comment:
How about 'We aim to push a new release of Prebid.js each week, barring unforeseen circumstances such as our main branch not passing tests or personal obligations."